features/gba-dev/overlijden-gba.feature

```diff
@@ -0,0 +1,231 @@
+    | aanduiding in onderzoek (83.10) | 060810                    |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    | datum overlijden (08.10)        | 19630405                  |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | overlijden.plaats               |
+    Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+    | naam                                         | waarde      |
+    | inOnderzoek.datumIngangOnderzoek             | 20020701    |
+    | inOnderzoek.aanduidingGegevensInOnderzoek    | 060810      |
+
+  Scenario: plaats overlijden is in onderzoek, maar wordt niet gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'overlijden' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | 060820                    |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    | plaats overlijden (08.20)       | 0518                      |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | overlijden.datum                |
+    Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+    | naam                                         | waarde      |
+    | inOnderzoek.datumIngangOnderzoek             | 20020701    |
+    | inOnderzoek.aanduidingGegevensInOnderzoek    | 060820      |
+
+  Scenario: land overlijden is in onderzoek, maar wordt niet gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'overlijden' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | 060830                    |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    | land overlijden (08.30)         | 6014                      |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | overlijden.plaats               |
+    Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+    | naam                                         | waarde      |
+    | inOnderzoek.datumIngangOnderzoek             | 20020701    |
+    | inOnderzoek.aanduidingGegevensInOnderzoek    | 060830      |
+
+  Rule: Onjuiste gegevens worden niet geleverd
```

Comment:
Ik begrijp deze rule eigenlijk niet. Na correctie zit de onjuiste gegeven in de rij met volg_nr 1. Voor actueel gegevens wordt rij met volg_nr 0 geretourneerd. Mis ik iets?